db/migrate/20210908203439_add_email_to_donations.rb

```diff
@@ -0,0 +1,6 @@
+class AddEmailToDonations < ActiveRecord::Migration[6.1]
+  def change
+    add_column :donations_payments, :email_status, :integer, limit: 1, default: 0, null: false
+    add_column :donations_subscriptions, :email_status, :integer, limit: 1, default: 0, null: false
```

Comment:
Will we retroactively send emails to people who donated?

Reply:
Yeah - I'm going to do that once we push this out.